card-games-vite/src/components/App/App.css

```diff
@@ -0,0 +1,18 @@
+.page {
+  background-color: #35654d;
+  font-family: "Roboto";
```

Comment:
It's important to always provide **fallback** and **generic** font name, like that: font-family: 'Inter', Arial, sans-serif; 
https://developer.mozilla.org/en/docs/Web/CSS/font-family

Font names are just for an example, you have your own font names and need to set fallback fonts based on it. For example, `Noto Serif` and `Inter` belong to different types, and they need different fallback fonts.
https://www.w3schools.com/css/css_font.asp 

Please, make this correction for **all the places** where you use `font-family`